src/tests/routes/controllers/password.test.js

```diff
@@ -0,0 +1,76 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import models from '../../../db/models';
+import app from '../../../server';
+
+// configure chai to use expect
+chai.use(chaiHttp);
+const { expect } = chai;
+
+before(async () => {
+  await models.sequelize.sync({ force: true });
+});
+let resetToken;
+const userEmail = {
+  email: 'jsmith@gmail.com',
+};
+
+describe('Forgot Password', () => {
+  it('should send the user a password reset link via email', (done) => {
+    chai.request(app)
+      .post('/api/v1/users/forgot-password')
+      .send(userEmail)
+      .end((err, res) => {
+        resetToken = res.body.data;
+        expect(res.status).to.equal(200);
+        expect(res.body.message).to.equal('Email has been sent successfully');
+        expect(res.body.data).to.be.a('string');
```

Comment:
Can you test for the data value instead of testing for the type of the response data.

Reply:
`res.body.data` is a token which will always be dynamic, it is called `data` due to the setup of the response helper function. Line `26` tests for data value.

Reply:
The data should not be token, you can pass in `{token: 'thetokenhere'}` as value of data, this way we can get the token as `res.data.token`.

Reply:
this has been resolved